stories/setup/config.js

```diff
@@ -5,6 +5,10 @@ import infoAddon, {
   setDefaults as setAddonInfoDefaults,
 } from '@storybook/addon-info';
 
+// Without core/polyfill we see the following error:
```

Comment:
You don't really need this comment. The polyfill is required for lots of things to work.